native-activity/app/build.gradle

```diff
@@ -5,38 +5,39 @@ model {
         compileSdkVersion = 23
         buildToolsVersion = "23.0.2"
 
-        defaultConfig.with {
+        defaultConfig {
             applicationId = "com.example.native_activity"
             minSdkVersion.apiLevel    = 9
             targetSdkVersion.apiLevel = 23
         }
-    }
-
-    android.ndk {
-        moduleName ="native-activity"
-        CFlags.add("-I${file("src/main/jni/native_app_glue")}".toString())
-        ldLibs.addAll(["log", "android", "EGL", "GLESv1_CM"])
-    }
-    android.buildTypes {
-        release {
-            minifyEnabled = false
-            proguardFiles.add(file('proguard-rules.txt'))
-        }
-    }
-    android.productFlavors {
-        create ("arm7") {
-            ndk.abiFilters.add("armeabi-v7a")
+        ndk {
+            stl = "gnustl_static"
+            moduleName ="native-activity"
+            cppFlags.add("-I${file("src/main/jni/native_app_glue")}".toString())
+            cppFlags.add("-std=c++11")
+            ldLibs.addAll(["log", "android", "EGL", "GLESv1_CM"])
         }
-        create ("arm8") {
-            ndk.abiFilters.add("arm64-v8a")
+        buildTypes {
+            release {
+                minifyEnabled = false
+                proguardFiles.add(file('proguard-rules.txt'))
+            }
         }
-        create ("x86-32") {
-            ndk.abiFilters.add("x86")
+        productFlavors {
+            create ("arm7") {
+                ndk.abiFilters.add("armeabi-v7a")
+            }
+            create ("arm8") {
+                ndk.abiFilters.add("arm64-v8a")
+            }
+            create ("x86-32") {
```

Comment:
certainly removing all is an option. but what lost is the capability that I build one flavor (ABI) in a apk and sends that one via usb to the target system.  when all deleted, android studio will tuck every ABI in debug/release build, not sure we want that